dnf/cli/cli.py

```diff
@@ -296,7 +296,13 @@ def latest_changelogs(self, package):
     def format_changelog(self, changelog):
         """Return changelog formated as in spec file"""
         chlog_str = '* %s %s\n%s\n' % (
-            changelog['timestamp'].strftime("%a %b %d %X %Y"),
+            # TRANSLATORS: This is the time format for changelog. You are
+            # encouraged to change it according to the requirements of your
+            # language. The format specifiers used here: %a - abbreviated
+            # weekday name in your language, %b - abbreviated month name in the
+            # correct grammatical form, %d - day number (00-31), %X - default
+            # time format for your language, %Y - year number (4 digits).
+            changelog['timestamp'].strftime(_("%a %b %d %X %Y")),
```

Comment:
I'm a bit afraid of this - incorrect format string in translation could cause a strange dnf behaviour in runtime. How about changing format string to plain "%c"?
See [https://docs.python.org/3/library/datetime.html#strftime-strptime-behavior](https://docs.python.org/3/library/datetime.html#strftime-strptime-behavior) :
`%c: Locale’s appropriate date and time representation.`

Reply:
Sounds good although ```strftime("%c")``` often contains the time zone (```"%Z"```) as well. You can use this command:
```bash
$ LC_TIME=cs_CZ.utf8 locale d_t_fmt
```
to see what string ```"%c"``` will be expanded to (replace ```cs_CZ``` with other locales). Personally I think it should not contain the time zone because it does not contain in the C locale but there is no standard specifying these details.

On the other hand, it may be good to assume that if ```"%c"``` contains the time zone then people really want this, otherwise they should speak up.

From my experience translators tend to leave the magic strings unchanged if they don't understand them, or they do understand and translate wisely. Also there is a comment for translators to explain the magic.

Are you still in favor of ```"%c"```? No problem if you are, I will follow your suggestion.

Reply:
Yes, I still would  go with %c.

Reply:
Done. I assume that ```"%c"``` does not need any translation.